template/package.json

```diff
@@ -103,6 +106,9 @@
     "postcss-import": "^11.0.0",
     "postcss-loader": "^2.0.8",
     "postcss-url": "^7.2.1",
+    {{#deploy}}
+    "push-dir": "^0.4.1",
```

Comment:
There's a package that makes this even easier imho: gh-pages

Have you considered it?

Reply:
I did consider the gh-pages package, but when checking out both of the packages documentation, gh-pages just seems to make it more complicated than it should be.

With the push-dir package, any flags can simply be added to the package.json file, such as --allow-unclean and --force, if required by the end-user.